KH2FM.cs

```diff
@@ -80,8 +87,9 @@ protected override void StartEffect(EffectRequest request)
 
     protected override bool StopEffect(EffectRequest request)
```

Comment:
Both in the UI, in game, and in the logs, I'm not seeing attempts to stop the effect when running the pack in the Crowd Control App. If I run the pack in the CrowdControlSDK and manually invoke stopping the effect, things work as expected. I'm wondering if there's something I'm missing with how effect durations are configured when the pack is initialized.

I've tried to do as much as I can to figure this out on my own but I think I need some help from the CC team to help get past this block.